### PR TITLE
Update LocalSettings.php

### PR DESCRIPTION
**Pull request by 1-2-1-2-3-Miraheze**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -2717,22 +2717,25 @@
...
 			),
 			'sysop' => array(
 				'moderator',
-			),
-			'bureaucrat' => array(
-				'moderator',
 				'confirmed',
 				'autopatrolled',
-				'rollbacker',
+				'rollback',
```
Comment (Reception123) [inline LocalSettings.php]: This should be rollbacker

File: LocalSettings.php [hunk 2]
```diff
@@ -2717,22 +2717,25 @@
...
+				'bot',
+			
+			),
+			'bureaucrat' => array(
```
Comment (Reception123) [inline LocalSettings.php]: Don't redefine bureaucrat. Add sysop to the previous list.

File: LocalSettings.php [hunk 3]
```diff
@@ -3320,23 +3323,25 @@
...
-			'bureaucrat' => array(
-				'moderator',
 				'confirmed',
 				'autopatrolled',
-				'rollback',
+				'rollbacker',
 				'bot',
+				
+			),
+			'bureaucrat' => array(
```
Comment (Reception123) [inline LocalSettings.php]: Same as above

Comment (1-2-1-2-3-Miraheze) [inline LocalSettings.php]: @Reception123 I don't know how to make changes to this file within the same PR - all changes you proposed above are approved by me to be made by someone else.

Comment (1-2-1-2-3-Miraheze) [inline LocalSettings.php]: I'm still trying to learn how to code directly as Phabricator is so slow. I'll get there eventually.

Comment (Reception123) [inline LocalSettings.php]: Also for editing you go on "Changes" and click the pencil button on the top right

**Comment by LegoFan4000** [comment]
Redone in #1058 per above

**Pull request closed without merging**